### ISSUE TITLE
Upload of new packages from travis to apt.dxtt.de fails with 406 Not Acceptable

### ISSUE THREAD

**Issue by klemens** [mswart/openmensa-parsers#89]
@mswart
The travis runs for the last three commits on master ([cedd712](https://travis-ci.org/mswart/openmensa-parsers/builds/466700882), [2eea58b](https://travis-ci.org/mswart/openmensa-parsers/builds/467173421), and [3fd2f37](https://travis-ci.org/mswart/openmensa-parsers/builds/467181917)) failed to upload the created package to apt.dxtt.de with the following error:
```
Resolving apt.dxtt.de (apt.dxtt.de)... 46.38.231.137, 2a03:4000:20:39::34
Connecting to apt.dxtt.de (apt.dxtt.de)|46.38.231.137|:80... connected.
HTTP request sent, awaiting response... 406 Not Acceptable
2018-12-12 20:07:41 ERROR 406: Not Acceptable.
```

**Comment by klemens** [comment]
I now have access to all parsers on openmensa.org, so I am now hosting the parsers at my own server: https://omfeeds.crpt.de

I have already switched over hannover, dresden, and marburg and @ialokim changed saarland. If everything works smoothly tomorrow, I will switch over the rest of the parsers.

This will also finally allow us to fix the [thueringen](https://github.com/mswart/openmensa-parsers/pull/79) (@Anarchoschnitzel) parser and add the new canteens to [wuerzburg](https://github.com/mswart/openmensa-parsers/pull/82) (@pstiegele) and [muenchen](https://github.com/mswart/openmensa-parsers/pull/85) (@FloEdelmann, @chneukirchen). Sorry again to all for the extremly long delays :disappointed:.

**Comment by klemens** [comment]
Everything seems to work fine with the new server! I switched over some more canteens: wuerzburg, muenchen, duesseldorf, halle (metadata currently broken)

I added the new canteens for wuerzburg and munechen and started preparing the ones for thueringen (not done yet, need to collect all addresses and geo locations).

I also added a [list](https://github.com/mswart/openmensa-parsers/wiki) to the wiki that contains links to all parsers on openmensa.org. Note that this is mainly useful for myself currently, as you cannot access these without being the owner or an admin.

**Comment by jgraichen** [comment]
@klemens You can access a parser as an administrator through the canteen page. There is a link "Parser öffnen" in the sidebar. It's not very elegant but best we have right now.

Maybe the parsers list can be managed by everyone (admin) on openmensa.org. Having every initial author maintaining their parsers alone seems to not work out.

**Comment by klemens** [comment]
@jgraichen: Yes, thats what I already did to create the list and update some parsers, no problems there.

If I have some time in the future and motivation to look into rails, I may try do some improvements in this regard.
